index.html

```diff
@@ -0,0 +1,571 @@
+  - [**Versioning.**](#versioning) Feature specifications follow [semver-like semantic versioning principles](#versioning), which helps schema processors determine if they are able to correctly interpret a document's metadata.
+  - [**Standard rules for extensibility**](#extensibility) of directives and their associated input types. These rules allow specs to annotate each others' metadata, effectively providing "directives on directives," which are otherwise impossible within GraphQL schemas
+
+**Core schemas are not a new language.** All core schema documents are valid GraphQL schema documents. However, this specification introduces new requirements, so not all valid GraphQL schemas are valid core schemas.
+
+The broad intention behind core schemas is to provide a *single document* which provides all the necessary configuration of a *data core*—some program which serves the schema to GraphQL clients, primarily by following directives in order to determine how to resolve queries made against that schema.
+
+### Parts of a Core Schema
+
+When talking about a core schema, we can broadly break it into two pieces:
+- an **export schema** (or **API**) consisting of exported schema elements (objects, interfaces, enums, directives, etc.) which **should** be served to clients, and
+- **machinery** consisting of directives and associated input types (such as enums and input objects). Machinery **must not** be served to clients.
+
+This reflects how core schemas are used: a core schema contains a GraphQL interface (the *API*) along with metadata about how to implement that interface (the *machinery*). Exposing the machinery to clients is unnecessary, and may in some cases constitute a security issue (for example, the machinery for a public-facing graph router will likely reference internal services, possibly exposing network internals which should not be visible to the general public).
+
+A key feature of core schemas is that it is always possible to derive the export schema from a core schema document without any knowledge of the features used by the document (with the exception of the `core` feature itself).
+
+Approximately, the process is:
+  - Any elements named `something__likeThis` are **not exported**, unless&#8230;
+    - &#8230;the feature which provides them ("`something`") has `export: true` on its `@core(feature:)` declaration, *or*
+    - &#8230;they are annotated with `@core__export` (or explicitly, `@core__export(isExport: true)`).
+  - Any elements with `normalNames` are **exported**, unless&#8230;
+    - &#8230;the element is annotated with `@core__export(isExport: false)`
+    - &#8230;the element is a directive or directive definition whose name matches the name of a feature, in which case, it is exported only if the entire feature is exported (having been brought in with `@core(feature:, export: true)`).
+
+A formal description is provided by the [is exported](#is-exported) algorithm.
+
+## Roles
```

Comment:
I'm not actually sure how useful this whole section is. It might be a bit more useful if it gives some non-normative examples of the different roles (eg, the federation composer as a machine Author). I'm not even sure we're planning to build any Processors any time soon, unless you count the composer?

May also be nice to include the export schema somehow connected to Data Core.